lib/src/reducers/strands_copy_info_reducer.dart

```diff
@@ -101,7 +102,23 @@ Tuple2<List<Strand>, List<int>> parse_strands_and_helices_view_order_from_clipbo
   if (helices_view_order_json != null) {
     helices_view_order = [for (int idx in helices_view_order_json) idx];
   }
-
+  
+  Map mod_jsons = clipboard_json[constants.design_modifications_key][0];
+  Map<String, Modification> mods = {};
+  for (var mod_key in mod_jsons.keys) {
+    Modification mod;
+    try {
+      mod = Modification.from_json(mod_jsons[mod_key]);
+    } on Exception {
+      print(error_msg);
```

Comment:
Hmm, here and elsewhere in this function, I find it odd that we print the error message and return null. Is this handled gracefully for this function to return null? If there's an error here let's try to make sure it doesn't do something super strange.

Reply:
It seems the function is under the impression it will return null on JSON decode error, so we've added a comment to let the user know to check for it when using this function. Is this alright?

Reply:
Looks good.